platformio.ini

```diff
@@ -83,6 +83,7 @@ build_flags               = -D BUILD_GIT='"${sysenv.TRAVIS_TAG}"'
                    -D PIO_FRAMEWORK_ARDUINO_LWIP2_LOW_MEMORY
                    -D NDEBUG
                    -D VTABLES_IN_FLASH
+                   -D USE_CUSTOM_H
```

Comment:
Do not include this in the platform.ini for everyone.
If you don't have the custom.h file, compilation will fail with this define set.